cmd/resim/commands/styling.go

```diff
@@ -44,10 +49,7 @@ var ReSimUsageTemplate string = `{{StyleHeading "USAGE"}}{{if .Runnable}}
   {{rpad .Name .NamePadding }} {{.Short}}{{end}}{{end}}{{end}}{{if not .AllChildCommandsHaveGroup}}
 
 {{StyleHeading "ADDITIONAL COMMANDS"}}{{range $cmds}}{{if (and (eq .GroupID "") (or .IsAvailableCommand (eq .Name "help")))}}
-  {{rpad .Name .NamePadding }} {{.Short}}{{end}}{{end}}{{end}}{{end}}{{end}}{{if .HasAvailableLocalFlags}}
-
-{{StyleHeading "FLAGS"}}
-{{.LocalFlags.FlagUsages | trimTrailingWhitespaces}}{{end}}{{if .HasAvailableInheritedFlags}}
+  {{rpad .Name .NamePadding }} {{.Short}}{{end}}{{end}}{{end}}{{end}}{{end}}{{if .HasAvailableLocalFlags}}{{FlagSetUsages .LocalFlags | trimTrailingWhitespaces}}{{end}}{{if .HasAvailableInheritedFlags}}
```

Comment:
q: how are you building these? by hand?

Reply:
https://github.com/spf13/cobra/blob/285460dca6152bb86994fa4a9659c24ca4060e2f/command.go#L539

Reply:
That's the hard-coded default. You can print it with something like `fmt.Print(cmd.UsageTemplate())`. I took that as a starting point and then began editing it.

Reply:
There's docs on how these templates work here: https://pkg.go.dev/text/template